src/data/monsters.txt

```diff
@@ -376,23 +376,23 @@ Knob Goblin Mutant	88	kg_mutant.gif	Atk: 53 Def: 47 HP: 43 Init: 70 Meat: 55 P:
 Knob Goblin poseur	533	kg_poseur.gif	Atk: 4 Def: 3 HP: 5 Init: 50 Meat: 20 P: goblin	dope gangsta bling-bling (10)	out-of-tune biwa (10)	terrible poem (20)
 Knob Goblin Sous Chef	76	kg_souschef.gif	Atk: 20 Def: 18 HP: 15 Init: 60 Meat: 20 P: goblin	Knob Goblin spatula (7)	Knob sausage (15)	tomato (15)	wad of dough (15)
 Knob Goblin Very Mad Scientist	86	kg_verymadsci.gif	Atk: 45 Def: 40 HP: 30 Init: 70 Meat: 45 P: goblin	Knob Goblin melon baller (5)	Knob Goblin seltzer (15)	Knob Goblin superseltzer (15)	scrumptious reagent (25)	bunch of square grapes (c19)	smudged alchemical recipe (p5)	Cobb's Knob Menagerie key (c0)	Subject 37 file (c0)
-Knott Slanding	617	slanding.gif	NOCOPY ULTRARARE NOMANUEL Atk: 150 Def: 135 HP: 150 Init: 10000 E: cold Meat: 970-1342 P: beast	Dallas Dynasty Falcon Crest shield (100)
-Knott Yeti	99	yeti.gif	Atk: 105 Def: 92 HP: 90 Init: 60 E: cold Meat: 200 P: beast	yeti fur (30)
+Knott Slanding	617	slanding.gif	NOCOPY NOMANUEL ULTRARARE Atk: 150 Def: 135 HP: 150 Init: 10000 Meat: 1156 P: beast E: cold	Dallas Dynasty Falcon Crest shield (100)
```

Comment:
I think meat should be left dashed in this case: it shows more clearly that these are values determined from fights, instead of from more thorough spading. I doubt the true value is 1156.

Reply:
Interesting. Again, the old code did the collapsing and passed a single integer to the MonsterData constructor.
Perhaps I should have Meat: have either an Integer or a String and the constructor does the collapsing.
I wonder how many monsters have this?

The number which is stored is the average. ASH proxy fields "min_meat" and "max_meat" are calculated: the stored value +- 20%

Again, this is an issue for going back from Attributes to String.

Reply:
I fixed this to calculate (and store) average meat from a range, but leave the range in the attribute.
Debug logging showed this only affects the Knott Slanding.
That's interesting, but we don't need to nag about this single monster, so I disabled the logging.